### PR TITLE
Picker enhancements

### PR DESCRIPTION
**Pull request by dannbeau**
- [x] Support custom status bar style to allow light content
- [x] Support custom tint colour for selected images
- [x] Add a `Done` bar button item to allow users to complete picking from albums list
- [x] Fixed "Moments" default string and use overridden moment string for title if set

### CODE REVIEW

File: NohanaImagePicker/AssetCell.swift [hunk 1]
```diff
@@ -32,6 +32,10 @@ class AssetCell: UICollectionViewCell {
 
             pickButton.setImage(droppedImage, for: UIControlState())
             pickButton.setImage(pickedImage, for: .selected)
+			
+			if let selectionTint = nohanaImagePickerController.config.color.selectionTint {
+				overlayView.backgroundColor = selectionTint
+			}
```
Comment (autoreleasefool) [inline NohanaImagePicker/AssetCell.swift:38]: spaces vs tabs 😂 the rest of the file uses spaces.... 😬

File: NohanaImagePicker/AssetCell.swift [hunk 2]
```diff
@@ -55,7 +59,10 @@ class AssetCell: UICollectionViewCell {
         self.asset = asset
         self.nohanaImagePickerController = nohanaImagePickerController
         self.pickButton.isSelected = nohanaImagePickerController.pickedAssetList.isPicked(asset)
+		if let selectionTint = nohanaImagePickerController.config.color.selectionTint {
+			overlayView.backgroundColor = selectionTint
+		}
```
Comment (autoreleasefool) [inline NohanaImagePicker/AssetCell.swift:64]: more tabs

File: NohanaImagePicker/NohanaImagePicker.storyboard [hunk 3]
```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (autoreleasefool) [inline NohanaImagePicker/NohanaImagePicker.storyboard:1]: storyboards 😭 I literally can't review this, I dunno how they work 🤷‍♂️  all those little x and y changes

**Review by autoreleasefool — APPROVED** [review]
Dunno if you wanna leave the spaces and tabs mixed, a hassle to change but would be nice to be consistent, and I don't think we can convert the developer's entire project to tabs 😂

Otherwise, changes look good 👍 will tophat in person?

File: NohanaImagePicker/AssetCell.swift (hunk 1 above)
Comment (dannbeau) [reply NohanaImagePicker/AssetCell.swift:38]: Yeah I think once we get this shipped, we should get this repo to be all in tabs.

I didn't want to bother matching it to spaces right now.

**Pull request merged**